Description: Relax constraints on dependency versions
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,7 +54,7 @@ name = "spdx"
 path = "src/lib.rs"
 
 [dependencies.smallvec]
-version = "1.15"
+version = "1"
 
 [dev-dependencies.similar-asserts]
-version = "1.7"
+version = "1"
